### ISSUE TITLE
"Error: Cannot find module './args'" error when first use

### ISSUE THREAD

**Issue by holmok** [Xstoudi/pino-pg#25]
implemented this way

```
nodemon index.js | pino-pg --connectionUrl postgresql://xxx:xxx@xxx:5432/holmok --table holmok_me.logs --column log | pino-tiny,
```

I installed the latest version an I am getting the following error:
```
Error: Cannot find module './args'
Require stack:
- /Users/christopher.holmok/work/holmok.me/node_modules/pino-pg/dist/src/index.js
- /Users/christopher.holmok/work/holmok.me/node_modules/pino-pg/index.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:980:15)
    at Function.Module._load (internal/modules/cjs/loader.js:862:27)
    at Module.require (internal/modules/cjs/loader.js:1040:19)
    at require (internal/modules/cjs/helpers.js:72:18)
    at Object.<anonymous> (/Users/christopher.holmok/work/holmok.me/node_modules/pino-pg/dist/src/index.js:9:32)
    at Module._compile (internal/modules/cjs/loader.js:1151:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1171:10)
    at Module.load (internal/modules/cjs/loader.js:1000:32)
    at Function.Module._load (internal/modules/cjs/loader.js:899:14)
    at Module.require (internal/modules/cjs/loader.js:1040:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/christopher.holmok/work/holmok.me/node_modules/pino-pg/dist/src/index.js',
    '/Users/christopher.holmok/work/holmok.me/node_modules/pino-pg/index.js'
  ]
}
```

**Comment by Xstoudi** [comment]
Hello,
Thanks for the report. It should be fixed in v2.1.3. Feel free to reopen the issue if needed.

**Comment by holmok** [comment]
Thanks for the quick fix.
